private/scripts/sage-theme-install.sh

```diff
@@ -227,24 +226,49 @@ function add_symlink() {
   fi
 
   # Create a files/cache directory on the host.
-  sftp -P 2222 $sftpuser@$sftphost <<EOF
+  sftp -P 2222 "$sftpuser"@"$sftphost" <<EOF
     cd /files
     mkdir cache
 EOF
 
     # Switch back to Git mode.
-    terminus connection:set $sitename.dev git
+    terminus connection:set "$sitename".dev git
 
   # Create the symlink to /files/cache.
-  cd web/app
+  cd web/app || return
   echo -e "${yellow}Adding a symlink to the cache directory.${normal}"
-  ln -sfn uploads/cache
+  ln -sfn uploads/cache .
```

Comment:
Not knowing the scripts use super-well, can you speak more to each of these changes?

Reply:
shellcheck flagged them as errors. that's it.